src/v/config/configuration.cc

```diff
@@ -1609,7 +1609,21 @@ configuration::configuration()
       "balancer, in milliseconds",
       {.needs_restart = needs_restart::no, .visibility = visibility::user},
       5000ms,
-      {.min = 1ms, .max = bottomless_token_bucket::max_width}) {}
+      {.min = 1ms, .max = bottomless_token_bucket::max_width})
+  , node_isolation_raft_timeout(
+      *this,
+      "health_monitor_append_entries_timeout_ms",
+      "Max time to waiting append_entries request before be isoalted",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      3000,
+      {.min = 100, .max = 10000})
```

Comment:
Since these properties are not std::optional, the only way someone can disable the behavior is to set a very high timeout.

Should either make them std::optional, or remove the upper bound on the timeout.

Reply:
To switch it user should use feature for isolation. Using timeout is not good idea

Reply:
I would like to use feature flags that way longer term, but today we don't have rpk support for that CLI, and we don't document feature flags externally, so the user won't know how to do it (whereas the cluster config schema is much more visible to them)